partiql-planner/src/main/kotlin/org/partiql/planner/internal/util/FunctionUtils.kt

```diff
@@ -1,4 +1,4 @@
-package org.partiql.planner.internal.utils
```

Comment:
(self-review) unified the `utils`/`util`/`helper` subpackages to be under `util`